test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,25 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        assertIsNotNull();
+        assertIsNotNull("abc");
+        assertIsNotNull("");
+        assertIsNotNull("abc123", "abcdef", "ghi");
+
+        assertIsNull((Object) null);
+        assertIsNull( null, null);
+    }
+
+    private void assertIsNull(Object... object) {
+        assertTrue(Utils.isAnyNull(object));
+    }
+
+    private void assertIsNotNull(Object... object) {
+        assertFalse(Utils.isAnyNull(object));
+    }
+
```

Comment:
Please use a clean PR for LO submissions as it would greatly improve the readability of your changes.